docs/addons/calico.md

```diff
@@ -1,16 +1,14 @@
-# Calico Add-on
+# Calico add-on
 
-[Project Calico](https://www.projectcalico.org/) is an open source networking and network security solution for containers, virtual machines, and native host-based workloads. To secure workloads in Kubernetes, Calico utilizes Network Policies. The `Calico` add-on adds support for Calico to an EKS cluster.
-
-By default, the native [VPC-CNI plugin](https://docs.aws.amazon.com/eks/latest/userguide/pod-networking.html) for Kubernetes on EKS does not support Kubernetes Network Policies. Installing Calico (or alternate CNI provider) will enable customers to define and apply standard [Kubernetes Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/) to their EKS cluster. 
```

Comment:
don't drop

Reply:
Added and revised section for VPC CNI plugin.